en/firefox/browsers/windows-64-bit.ftl

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/windows-64-bit/
+
+
+# HTML page title
+windows-64-bit-firefox-for-windows = { -brand-name-firefox } for { -brand-name-windows } 64-bit
+
+# HTML page description
+windows-64-bit-users-on-64-bit-windows = Users on 64-bit { -brand-name-windows } who download { -brand-name-firefox } can get our 64-bit version by default. That means you get a more secure version of { -brand-name-firefox }.
+windows-64-bit-64-bit = 64-bit
+
+# Main title
+windows-64-bit-a-more-secure-firefox = A more secure { -brand-name-firefox }.
+
+# Variables:
+# $crashes (url) - link to https://blog.mozilla.org/firefox/defeat-browser-crashes/
+windows-64-bit-users-on-64-bit-windows-that-means = Users on 64-bit { -brand-name-windows } who download { -brand-name-firefox } can get our 64-bit version by default. That means you get a more secure version of { -brand-name-firefox }, one that also <a href="{ $crashes }">crashes a whole lot less</a>. How much less? In our tests so far, 64-bit { -brand-name-firefox } reduced crashes by 39% on machines with 4GB of RAM or more.
+windows-64-bit-whats-the-difference = What’s the difference between 32-bit and 64-bit?
+
+# Variables:
+# $ASLR (url) - link to https://en.wikipedia.org/wiki/Address_space_layout_randomization
+windows-64-bit-heres-the-key-thing = Here’s the key thing to know: 64-bit applications can access more memory and are less likely to crash than 32-bit applications. Also, with the jump from 32 to 64 bits, a security feature called <a href={ $ASLR }>Address Space Layout Randomization (ASLR)</a> works better to protect you from attackers. { -brand-name-linux } and { -brand-name-mac } users, fret not, you already enjoy a { -brand-name-firefox } that’s optimized for 64-bit.
```

Comment:
Note: this string is missing quotes around the URL (it's `href=x` when it should be `href="x"`). I'd like to fix that, but I'm sure that will break things when I run the script to convert the .lang files. I'm prepared to go through all the Fluent files one by one and fix them manually if I need to, but that will have to happen after the initial migration.

Reply:
I don't quite understand why this one is different from the other URLs that have the correct format, but that's ok. Looks good otherwise!

Reply:
Probably just a mistake when the page was coded. It still works fine, I just want to fix it while I have the chance.